webpack/assets/javascripts/react_app/components/common/Alert/AlertBody.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AlertLink from './AlertLink';
+
+const AlertBody = ({
+  link, title, message, children,
+}) => (
+  <span>
+    {link && <AlertLink {...link} />}
+
+    {title && <strong>{title}</strong>}
+
+    <span dangerouslySetInnerHTML={{ __html: message }} />
+
+    {children}
```

Comment:
To support in overflowed text you could wrap `children / message` with `react-ellipsis-with-tooltip' :)

Reply:
I afraid it will cause some problems because rails is writing the notifications into the dom, then our javascript stack will read it from the dom, transform the data into toasts and then remove the data from the dom.

About the `react-ellipsis-with-tooltip`, i don't want to force this behavior, currently, when there is no space for the text you will get 2 rows of text which looks good.
We can use the `react-ellipsis-with-tooltip` only when we want to cancel this behavior in specific places.